toolchain/static-link.lisp

```diff
@@ -33,6 +33,18 @@
 
 ;; Problem? Its output may conflict with the program-op output :-/
 
+(defmethod perform :before ((op static-image-op) (s system))
+  ;; Close non-system foreign libraries, as grovel ones are embedded
+  ;; in the static-image. System foreign libraries still need to be
+  ;; explicitly loaded on startup, so we're keeping them loaded.
+  (register-image-dump-hook
+   (lambda ()
+     (loop
+        for library in (list-foreign-libraries)
```

Comment:
Minor style suggestion, reformat as

```
(loop for library in (list-foreign-libraries)
      when (eql ...)
        do (close ...)) 
```

do you have slime-indentation enabled?

Or just use dolist, that'd work too.

Reply:
dolist sounds good, using that.

Reply:
Pushed an update with that.